tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/EncryptionTestHelper.cs

```diff
@@ -78,6 +78,17 @@ public static void ConfigureDefaultExtraOptions(Dictionary<string, object> extra
                     extraOptions.Add("mongocryptdSpawnArgs", portValue);
                 }
             }
+
+            if (!CoreTestConfiguration.GetTestMongdocryptd())
```

Comment:
nit: in other places here we use a `Environment.GetEnvironmentVariable`, maybe it's better to not introduce a helper until it's actually needed

Reply:
Done.